### PR TITLE
Logs layout changes

### PR DESCRIPTION
**Pull request by ijunaid8989**
/cc @azharmalik3

### CODE REVIEW

File: app/assets/stylesheets/views/cameras/single/_logs.scss [hunk 1]
```diff
@@ -191,3 +191,29 @@
...
+
+.checkbox-list > div.input-group:first-child {
+ margin-left: 0px;
+}
+
+.min-height-67 {
+  min-height: 67px;
+}
+
+#logs-table_length {
```
Comment (ijunaid8989) [inline app/assets/stylesheets/views/cameras/single/_logs.scss]: This is auto generated id by data tables.

**Comment by ijunaid8989** [comment]
The new logs layout changes look like that... 
selection_150
Please review the PR @azharmalik3

**Comment by azharmalik3** [comment]
:+1:

**Comment by marcoherbst** [comment]
@ijunaid8989 ?

**Pull request merged**